db/seeds.rb

```diff
@@ -26,14 +26,14 @@ def random_record_for_org(org, klass)
 
 # Initial starting qty for our test organizations
 base_items = File.read(Rails.root.join("db", "base_items.json"))
-items_by_category = JSON.parse(base_items)
+@items_by_category = JSON.parse(base_items)
```

Comment:
Is there a reason to make this an instance variable instead of a plain variable? (not a rhetorical question; I'm just unclear on this change)

Reply:
Yes. I needed instance variables when I moved code that accessed them into methods.